docker.js

```diff
@@ -686,6 +686,10 @@ export const getImage = async (fullImageName) => {
         return undefined;
       }
     } catch (err) {
+      // This style of retry is timing out with gcr so disabled for now.
+      if (registry?.startsWith("gcr.io")) {
```

Comment:
ghcr.io?

Reply:
google container registry. I might remove this eventually, so have marked this PR as draft.